app/views/visitors/index.html.erb

```diff
@@ -19,10 +19,37 @@
           <li>Earth's moon</li>
           <li>Pluto</li>
         </ul>
+      </article>
+    </row>
+  </section>
 
-      <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+  <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+
+  <section class="container requirements">
+    <div class="row">
+      <article class="col">
+        <h2 class="govuk-heading-l">What you will need ready</h2>
+        <p class="govuk-body">
+          To apply for a landing permit you must provide your:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>name</li>
+          <li>email address</li>
+          <li>date of birth</li>
+          <li>passport number</li>
+          <li>pilot's licence number</li>
+          <li>vehicle registration number</li>
+        </ul>
+        <p class="govuk-body">You will also need to tell us the dates of your proposed visit as not all dates are available on all destinations, due to local public holidays.</p>
```

Comment:
Maybe not the priority while prototyping, but this line is too long on medium-large screens so we should include something like this to constrain the width of textual elements for accessibility:

```html
<div class="govuk-grid-row">
    <div class="govuk-grid-column-two-thirds">
```

Reply:
Good point, thanks. I've added this in a separate commit.